charts/piraeus/crds/piraeus.linbit.com_linstorcsidrivers_crd.yaml

```diff
@@ -57,6 +57,17 @@ spec:
             linstorControllerAddress:
               description: Cluster address of the linstor controller
               type: string
+            linstorHttpsClientSecret:
```

Comment:
I don't see this being set anywhere in the Helm chart yet.

Reply:
Fixed